helm-chart-sources/edge/templates/daemonset.yaml

```diff
@@ -105,6 +105,12 @@ spec:
                   key: CRIBL_DIST_LEADER_URL
             - name: CRIBL_HOME
               value: {{ .Values.cribl.home }}
+            # CPU Limit
+            - name: CRIBL_KUBE_CPU_LIMIT
+              valueFrom:
+                resourceFieldRef:
+                  container: {{ .Chart.Name }}
```

Comment:
Do we expect the container name to always match the chart name?

Reply:
it sets the container name further up in the spec using the same thing but it seems to work without specifying the container name there at all so I'm inclined to just remove it.